xivlauncher: 1.0.2 -> 1.0.3

witchof0x20 opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

keysmashes reviewed (approved):
lgtm (verified locally no deps changes)

Result of `nixpkgs-review pr 210737` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>xivlauncher</li>
  </ul>
</details>

probably worth removing `ashkitten` from maintainers (per request on discord), and adding yourself if you want

winterqt commented on pkgs/games/xivlauncher/default.nix:
```diff
@@ -13,7 +13,7 @@ in
       repo = "XIVLauncher.Core";
       inherit rev;
-      sha256 = "DlSMxIbgzL5cy+A5nm7ZaA2A0TdINtq2GHW27uxORKI=";
+      sha256 = "aQVfW6Ef8X6L6hBEOCY/Py5tEyorXqtOO3v70mD7efA=";
```
Why is the bare base-64 hash used? This would probably be better off using SRI:

```suggestion
      hash = "sha256-aQVfW6Ef8X6L6hBEOCY/Py5tEyorXqtOO3v70mD7efA=";
```

witchof0x20 replied:
Was just keeping consistent with the previous file. updated!

sersorrel replied:
("nix-prefetch-github prints the bare base64 hash" is why it was like that originally, fwiw. maybe i'll try and get a PR upstream to either make the hash used configurable or to just include the `sha256-` prefix by default)

winterqt replied:
Ah, makes sense. Not really sure if it's worth changing then, but doesn't really matter either way.

winterqt commented:
> probably worth removing `ashkitten` from maintainers (per request on discord)

@sersorrel Can you have them submit a PR themselves, if possible?

ashkitten commented:
@winterqt i confirm i asked to be removed as maintainer, if that helps. i can submit my own pr if you still want that.

witchof0x20 committed:
xivlauncher: 1.0.2 -> 1.0.3

winterqt commented:
> @winterqt i confirm i asked to be removed as maintainer, if that helps. i can submit my own pr if you still want that.

I'll push it here, if you don't mind.

@witchof0x20 This all needs to be collapsed into 2 commits:

```
maintainers: add witchof0x20
xivlauncher: add witchof0x20 as maintainer
xivlauncher: 1.0.2 -> 1.0.3
```

witchof0x20 commented:
Done!

winterqt committed:
xivlauncher: remove ashkitten as maintainer

Requested in https://github.com/NixOS/nixpkgs/pull/210737#issuecomment-1382894954

winterqt commented:
@witchof0x20 @ashkitten @sersorrel Everything look good?

witchof0x20 commented:
LGTM

ashkitten commented:
lgtm

sersorrel commented:
maintainer-list.nix:
>     When editing this file:
>     * keep the list alphabetically sorted

but otherwise lgtm (and frankly it's not sorted already so)

winterqt commented:
Ah, assumed it was... oops. @witchof0x20 can you fix that, please? I'll fix the other entries.

witchof0x20 commented:
Fixed!

witchof0x20 committed:
maintainers: add witchof0x20 as maintainer

witchof0x20 committed:
xivlauncher: add witchof0x20 as maintainer

witchof0x20 commented:
Oh, for whatever reason my change commit didnt go through when rearranging it <.<
Should be all up to par now, sorry!
